shared/fs/footer/upload-container.js

```diff
@@ -33,16 +34,22 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   debugToggleShow: getDebugToggleShow(dispatch),
 })
 
-const mergeProps = ({_uploads}, {debugToggleShow}) =>
-  ({
+const mergeProps = ({_uploads, _pathItems}, {debugToggleShow}) => {
+  const folders = _pathItems.filter(path => path.type === 'folder')
+  const uploadPaths = _uploads.syncingPaths // get array of paths
```

Comment:
Hmm why reassigning it to a different name? Also, you can check [this file](https://github.com/keybase/client/blob/master/shared/constants/types/fs.js) for types. `syncingPaths` is an `I.Set` here.